backend/server/config.py

```diff
@@ -2,10 +2,10 @@
 
 URI = "mongodb://mongodb:27017/?readPreference=primary&appname=MongoDB%20Compass&directConnection=true&ssl=false"
 
-CLIENT_ID = "1017197944285-i4ov50aak72667j31tuieffd8o2vd5md.apps.googleusercontent.com"
-
 FINAL_DATA_PATH = "./data/final_data/"
 
 ARCHIVED_DATA_PATH = "./data/final_data/archive/processed/"
 
-DUMMY_TOKEN = "token"
+DUMMY_TOKEN = "token"  # TODO: get rid of this
```

Comment:
yaaa dummy data like bad_planner is used a lot in frontend because we didnt make a unified error or loading display. 
in my react query followon PR, i will likely unify it all.

for dummy_token, its only used now in testing, so when we fix tests, we can just use guest login route instead.